### PR TITLE
Fix sitemap only show correct domains

### LINKED ISSUE #991: Sitemap not valid for Google
Sitemap isn't valid for Google if different domains occur in the same xml. In the example below the entry domain.fr/demande. With subdomains it would be valid for Google (e.g. domain.ch/anfrage and fr.domain.ch/demande could be in the same xml file)

Example:

This is the webspace config:
```xml
...
<environment type="prod">
    <urls>
        <url language="de">www.domain.ch</url>
        <url language="fr">www.domain.fr</url>
        <url language="it">www.domain.it</url>
    </urls>
</environment>
...
```

This is from the **www.domain.ch/sitemap.xml**:
```xml
...
<url>
    <loc>http://www.domain.ch/anfrage</loc>
    <lastmod>2015-01-12</lastmod>
</url>
<url>
    <loc>http://www.domain.fr/demande</loc>
    <lastmod>2015-01-12</lastmod>
</url>
...
```

### PR DESCRIPTION
**Pull request by alexander-schranz**
__Tasks:__

- [ ] test coverage
- [x] gather feedback for my changes
- [ ] submit changes to the documentation
- [x] added changelog line

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | yes
| Fixed tickets | fixes #991 
| BC Breaks     | none
| Doc           | none

### CODE REVIEW

**Comment by chirimoya** [comment]
@alexander-schranz ... can you please split this PR and remove all the multi-webspace-sitemap stuff e.g. `getLowestLevelDomainByDomain`. I don't feel sure about the current implementation of this addition feature.

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function getFunctions()
     /**
      * {@inheritdoc}
      */
-    public function getContentPath($url, $webspaceKey = null, $locale = null)
+    public function getContentPath($url, $webspaceKey = null, $locale = null, $domain = null)
```
Comment (chirimoya) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: Update interface as well.

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php [hunk 2]
```diff
@@ -77,9 +77,24 @@ public function getContentPath($url, $webspaceKey = null, $locale = null)
                 $locale ?: $this->requestAnalyzer->getCurrentLocalization()->getLocalization(),
                 $webspaceKey
             );
-
             if (sizeof($portalUrls) > 0) {
-                return rtrim($portalUrls[0], '/');
+                if ($domain) {
```
Comment (chirimoya) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: Not sure if the twig-extension is the right place to validate the sitemap urls. This extension is heavily used in other places. @danrot WDYT?

Comment (danrot) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: I think the same, this logic should probably also be useable in a non-twig context. Don't really know what you want to say with the usage...

Comment (alexander-schranz) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: moved into webspacemanager

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php (hunk 1 above)
Comment (alexander-schranz) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: done

**Comment by alexander-schranz** [comment]
@chirimoya multi webspace support removed

**Pull request merged**